<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="StuCheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>学生</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="TeaCheckBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>教职工</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="AdminCheckBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>管理员</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usrLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pswLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="loginBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
